grails-app/domain/org/pih/warehouse/core/Location.groovy

```diff
@@ -93,6 +93,13 @@ class Location implements Comparable<Location>, java.io.Serializable {
         dateCreated(display: false)
         lastUpdated(display: false)
         sortOrder(nullable: true)
+        supportedActivities(validator: { Set<String> activities, Location location ->
```

Comment:
I'm having trouble imagining how this is going to work. Does this handle the case of supported activities inherited from location type? For example, in that case the location's supported activities would be empty but the location type support activities would not be.

Reply:
If you "inherit" from the location type, your (location's) supported activities are empty.
This validator just checks if you try to add `None` together with any other activities, so it blocks having e.g.: `[None, Submit request]`, but allows: `[None]` or any combination without none, so: `[Submit request, Manage Inventory, Hold Stock]`